zjit/src/hir.rs

```diff
@@ -467,6 +467,42 @@ pub enum SideExitReason {
     StackOverflow,
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum SendWithoutBlockDefType {
```

Comment:
It's not really about `send_without_block` (this enum itself should be used for `send` too, for example), so maybe just call it `MethodType`, given it's called `rb_method_type_t` in C?